ush/rocoto/rocoto.py

```diff
@@ -68,6 +68,7 @@ def create_task(task_dict):
     jobname = task_dict.get('jobname', 'demojob')
     account = task_dict.get('account', 'batch')
     queue = task_dict.get('queue', 'debug')
+    nodesize = task_dict.get('nodesize', '128')
```

Comment:
Can we set this default to 1?

Reply:
Not sure it matters. This branch is only going to be used for ops.

Reply:
Changed default to 1, thanks!